### DISCUSSION TITLE
Hasura Open Source risk?

### DISCUSSION THREAD

**Discussion by osseonews** [nhost/nhost#2349] [General]
I'm wondering what will happen with Nhost if Hasura ever decides to abandon the open source? I ask because Hasura has taken some steps in the last year which indicate they don't care about the open source. For example, their new pricing is simply outrageous (reason to use nHost btw), and then recently I saw that Digital Ocean no longer has their image in the droplet marketplace. Looks to me like betting on Hasura is very risky. As nhost is so dependent on Hasura open source, what can be done to mitigate that risk? Thanks.

**Comment by dbarrosop** [comment]
That would depend on their move. But you can't change the LICENSE retroactively which means that what's opensource today will be opensource forever. In any case, next steps would depend on the nature of the change. Worst case scenario, a fork is always a possibility.
